test/ibmq/test_experiment.py

```diff
@@ -406,6 +406,7 @@ def test_get_analysis_results(self):
         results = self.provider.experiment.analysis_results()
         for res in results:
             self.assertTrue(isinstance(res, AnalysisResult))
+            self.assertEqual(res.verified, False)
```

Comment:
```suggestion
            self.assertFalse(res.verified)
```

Just a nit.

Reply:
Nope, but that test was testing the default value so I just removed it from the call to the constructor and let it's do it's thing internally.